util/src/main/java/org/folio/util/PostgresTester.java

```diff
@@ -9,7 +10,7 @@ public interface PostgresTester extends Closeable {
    * @param username Postgres username
    * @param password Postgres password
    */
-  void start(String database, String username, String password);
+  void start(String database, String username, String password) throws PostgresTesterStartException;
```

Comment:
I added `PostgresTesterStartException` because of a sonar to consolidate `IOException` and `InterruptedException` which were needed because of my use of `execInContainer` but your branch doesn't use `execInContainer` so there's no need for any `throws` items on the method.